content/en/blog/2025/sig-etcd-spotlight/sig-etcd-spotlight.md

```diff
@@ -0,0 +1,218 @@
+
+**Frederico: On top of those, what would you single out as the major benefit that has resulted from
+becoming a SIG?**
+
+**Marek**: The biggest benefits for me was adopting Kubernetes testing infrastructure, tools like
+[Prow](https://docs.prow.k8s.io/) and [TestGrid](https://testgrid.k8s.io/). For large projects like
+etcd there is just no comparison to the default GitHub tooling. Having known, easy to use, clear
+tools is a major boost to the etcd as it makes it much easier for Kubernetes contributors to also
+help etcd.
+
+**Wenjia**: Totally agree, while challenges remain, the SIG structure provides a solid foundation
+for addressing them and ensuring etcd's continued success as a critical component of the Kubernetes
+ecosystem.
+
+The positive impact on the community is another crucial aspect of SIG etcd's success that I’d like
+to highlight. The Kubernetes SIG structure has created a welcoming environment for etcd
+contributors, leading to increased participation from the broader Kubernetes community.  We have had
+greater collaboration with other SIGs like [SIG API
+Machinery](https://github.com/kubernetes/community/blob/master/sig-api-machinery/README.md), 
+[SIG Scalability](https://github.com/kubernetes/community/tree/master/sig-scalability), 
+[SIG Testing](https://github.com/kubernetes/community/tree/master/sig-scalability), 
+[SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle), etc. 
+
+This collaboration helps ensure etcd's development aligns with the needs of the wider Kubernetes
+ecosystem. The formation of the [etcd Operator Working Group]/(https://github.com/kubernetes/community/blob/master/wg-etcd-operator/README.md) 
+under the joint effort between SIG etcd and SIG Cluster Lifecycle exemplifies this successful
+collaboration, demonstrating a shared commitment to improving etcd's operational aspects within
+Kubernetes.
+
+**Frederico: Since you mentioned collaboration, have you seen changes in terms of contributors and
+community involvement in recent months?**
+
+**James**: Yes -- as showing in our 
+[unique PR author data](https://etcd.devstats.cncf.io/d/23/prs-authors-repository-groups?orgId=1&var-period=m&var-repogroup_name=All&from=now-10y&to=now-38d)
+we recently hit an all time high in March and are trending in a positive direction:
+
+<figure>
+	<img src="/blog/2025/sig-etcd-spotlight/stats.png" alt="Unique PR author data stats"></img>
+</figure>
+
+Additionally, looking at our 
+[overall contributions across all etcd project repositories](https://etcd.devstats.cncf.io/d/74/contributions-chart?orgId=1&from=now-10y&to=now-38d&var-period=m&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-company_name=All&var-company=all) 
+we are also observing a positive trend showing a resurgence in etcd project activity:
+
+<figure>
+	<img src="/blog/2025/sig-etcd-spotlight/stats2.png" alt="Overall contributions stats"></img>
+</figure>
```

Comment:
This should use a [figure](https://gohugo.io/content-management/shortcodes/#figure) shortcode, for ease of site maintenance.

Reply:
Done.